src/server/job_func.c

```diff
@@ -357,6 +357,7 @@ job_alloc(void)
 	CLEAR_HEAD(pj->ji_multinodejobs);
 	pj->ji_extended.ji_ext.ji_stdout = 0;
 	pj->ji_extended.ji_ext.ji_stderr = 0;
+	pj->ji_spawninfo = NULL;
```

Comment:
Since we are setting every field member of job by hand, should we remove the memset on top. That just seems to be wasteful

Reply:
Interesting question.  I took a quick look to see if I could find any fields from the job structure that were not specifically listed in in job_alloc.  By looking at the first 10-15 fields there were already a handful that were relying on the memset (i.e. not listed specifically in job_alloc()).  I would not recommend removing memset without a deeper dive.

Reply:
hmm - then there are certainly fields which we are setting to 0, even if memset did the job. We should delete those lines - small, but when we want thousands of job per second, every nanosecond counts :) That said, this is a general thought, not for this PR, so you may ignore it.